src/i18n/faucet_en.json

```diff
@@ -0,0 +1,24 @@
+{
+  "faucet": {
+    "common": {
+      "error": "Error",
+      "success": "Success"
+    },
+    "sendMeToken": "Send me AXONE",
+    "requestFundsWithKeplr": "Request funds with Keplr",
+    "requestFundsWithAddress": "Request funds with AXONE chain address",
+    "addressPlaceholder": "axonexxxxxxxxxxxxxxxxxxxxxxxxxxx",
+    "operationSucceeded": "Your transaction has been successfully sent!",
+    "brand": {
+      "title": "AXONE Faucet",
+      "description": "Hello fellow data space invader! Use this faucet to get tokens for the {{env}} chain. Please don't abuse this service—the number of available tokens is limited."
```

Comment:
I think we can update the catchphrase, because data spaces are no more relevant. For instance, as a suggestion:

```suggestion
      "description": "Greetings, web3 explorer! Claim your AXONE token for the {{env}} chain to kickstart your journey in the Axone ecosystem. Remember, this is a shared resource—the availability of tokens is limited."
```